tests/validate-dirty-db.py

```diff
@@ -56,6 +59,7 @@ def runNodeosAndGetOutput(myTimeout=3):
         proc.send_signal(signal.SIGKILL)
         return (False, None)
 
+    if debug: Print("Returning success status.")
```

Comment:
"success status" sounds like a generic thing, can we just make it "Returning success"?